pkg/reconciler/delivery/delivery.go

```diff
@@ -140,6 +138,18 @@ func (c *Reconciler) fetchRevisions(cfg *v1.Configuration) (map[string]*v1.Revis
 	return revisionMap, nil
 }
 
+// fetchPolicy queries the indexer to retrieve a Policy object and return its translated version
+// if annotations don't specify a Policy or if the specified Policy cannot be found, an error is returned
+func (c *Reconciler) fetchPolicy(cfg *v1.Configuration) (*Policy, error) {
+	// there's no need for defensive map query check, because it would have been taken care of by lines 89-92
```

Comment:
Remove line numbers in this comment, they are guaranteed to change.

Reply:
will do